src/api/TokenIO.h

```diff
@@ -170,6 +170,39 @@ globalRpcErrorCallback:(OnError)globalRpcErrorCallback;
                           onSuccess:(OnSuccess)onSuccess
                             onError:(OnError)onError;
 
+#pragma mark - Recovery
+
+/**
+ * Begins recovery process for an alias. The verification message will be sent if the alias is valid.
+ *
+ * @param aliasValue alias value to recover
+ * @param onSuccess invoked if successful
+ * @param onError invoked if failed
+ */
+- (void)beginRecovery:(NSString *)aliasValue
+            onSuccess:(OnSuccess)onSuccess
+              onError:(OnError)onError;
+
+/**
+ * Verifies recovery code after beginRecovery is successful.
+ *
+ * @param code code from verification message
+ * @param onSuccess invoked if successful
+ * @param onError invoked if failed
+ */
+- (void)verifyRecoveryCode:(NSString *)code
+                 onSuccess:(OnSuccessWithBoolean)onSuccess
+                   onError:(OnError)onError;
+
+/**
+ * Completes recovery process after verifyRecoveryCode is successful.
```

Comment:
I was surprised that our flow had a step after "verify". I had to peek at the source to see its purpose. Probably good to add something to the comment about what's in this step.

Maybe "Uploads member's public keys from this device to Token directory." ? (Yes, it's more complex than that. Not sure how much of that complexity to put into a header comment?)